File: backend/src/models/usuario.py
from dataclasses import dataclass

@dataclass
class Usuario:
    mail: str
    hash_contrasena: str

    codigo_pais_documento: str
    id_tipo_documento: int
    numero_documento: str

    codigo_pais_residencia: str
    localidad: str
    calle: str
    numero_puerta: str
    codigo_postal: str | None

File: frontend/src/types/usuario.ts
export type Usuario = {
  mail: string;
  contrasena: string;
  codigo_pais_documento: string;
  id_tipo_documento: number;
  numero_documento: string;
  codigo_pais_residencia: string;
  localidad: string;
  calle: string;
  numero_puerta: string;
  codigo_postal?: string;
  telefonos: string[];
};
